### ISSUE TITLE
Admin cannot change the owner (Owner) value of a record, once it has been set (even if he's the owner)

### ISSUE THREAD

**Issue by Vadorequest** [directus/app#2711]
How can I change the `owner` value once it has been set? I'm both admin and the owner of a record and I can't change the `owner` myself.

My workflow is to create the “Customer” record myself, not from a customer account (which doesn’t have such permission), and thus the “owner” is myself. I’d like to change it afterwards (most likely once I’ve created the customer’s role so that they may edit some of their own “Customer” record’s fields.

> The owner interface is always locked to the person who created the item. If you use a regular user interface, but set it's directus type to owner, you can set it to whatever owner you want, and that will be used for permissions. - Rijk's workaround

But this begs the question, shouldn't an admin be able to change the owner value of a record? I feel like it's not currently supported unless using a very particular setup.

**Comment by benhaynes** [comment]
To achieve better clarity/visibility, we are now tracking feature requests within the [Feature Request](https://github.com/directus/api/projects/40) project board.

**This issue being closed does not mean it's not being considered.**

**Comment by benhaynes** [comment]
Yes, and Admin should be able to change this — @rijkvanzanten, depending on how complicated this is, it'd be good to see if it can fit into v9.

**Comment by Vadorequest** [comment]
Indeed, if it's not too complicated then the sooner the better, because it changes quite a few things on the model design and that's not something trivial to change afterwards.

For now, I've used @rijkvanzanten trick and it works fine, but I've had to delete all the Owner fields in all my models, and use a User field with Owner type instead. One can only do that at the very beginning, I wouldn't know how to migrate the data once there are any real data stored.

And still, this design doesn't allow me to have proper accountability (multiple Owner fields in the same model, one that can't change and one that can) See https://github.com/directus/app/issues/2717
